docs/testplan/Smartswitch-test-plan.md

```diff
@@ -0,0 +1,876 @@
+ *  The "show reboot-cause history" CLI on the switch shows the history of the Switch and all DPUs
+ *  The "show reboot-cause history module-name" CLI on the switch shows the history of the specified module
+ *  Use `config chassis modules shutdown <DPU_Number>` 
+ *  Use `config chassis modules startup <DPU_Number>`
+ *  Wait for 5 minutes for Pmon to update the dpu states
+ *  Use `show reboot-cause <DPU_Number>` to check the latest reboot is displayed 
+   
+#### Verify in
+ * Switch
+   
+#### Sample Output
+```
+On Switch:
+
+root@sonic:~#show reboot-cause
+
+Device          Name                    Cause                       Time                                User    Comment
+
+switch          2023_10_20_18_52_28     Watchdog:1 expired;         Wed 20 Oct 2023 06:52:28 PM UTC     N/A     N/A
+DPU3            2023_10_03_18_23_46     Watchdog: stage 1 expired;  Mon 03 Oct 2023 06:23:46 PM UTC     N/A     N/A
+DPU2            2023_10_02_17_20_46     reboot                      Sun 02 Oct 2023 05:20:46 PM UTC     admin   User issued 'reboot'
+
+root@sonic:~#show reboot-cause history
+
+Device          Name                    Cause                       Time                                User    Comment
+
+switch          2023_10_20_18_52_28     Watchdog:1 expired;         Wed 20 Oct 2023 06:52:28 PM UTC     N/A     N/A
+switch          2023_10_05_18_23_46     reboot                      Wed 05 Oct 2023 06:23:46 PM UTC     user    N/A
+DPU3            2023_10_03_18_23_46     Watchdog: stage 1 expired;  Mon 03 Oct 2023 06:23:46 PM UTC     N/A     N/A
+DPU3            2023_10_02_18_23_46     Host Power-cycle            Sun 02 Oct 2023 06:23:46 PM UTC     N/A     Host lost DPU
+DPU3            2023_10_02_17_23_46     Host Reset DPU              Sun 02 Oct 2023 05:23:46 PM UTC     N/A     N/A
+DPU2            2023_10_02_17_20_46     reboot                      Sun 02 Oct 2023 05:20:46 PM UTC     admin   User issued 'reboot'
+
+"show reboot-cause history module-name"
+
+root@sonic:~#show reboot-cause history dpu3
+
+Device      Name                    Cause                           Time                                User    Comment 
+   
+DPU3        2023_10_03_18_23_46     Watchdog: stage 1 expired;      Mon 03 Oct 2023 06:23:46 PM UTC     N/A     N/A
+DPU3        2023_10_02_18_23_46     Host Power-cycle                Sun 02 Oct 2023 06:23:46 PM UTC     N/A     Host lost DPU
+DPU3        2023_10_02_17_23_46     Host Reset DPU                  Sun 02 Oct 2023 05:23:46 PM UTC     N/A     N/A
+```
+
+#### Pass/Fail Criteria 
+ * Verify the output to check the latest reboot cause with the date time stamp at the start of reboot
+ * Verify all the reboot causes - Watchdog, reboot command, Host Reset
+
+
+### 1.12 Check the DPU state after OS reboot
+
+#### Steps
+
+Existing Test case:
+   * Reboot using a particular command (sonic reboot, watchdog reboot, etc) (timeout 5 mins, wait 2 mins)
```

Comment:
The boot time may differ from vendor to vendor. This information should be part of the platform.json  file. Please check https://github.com/sonic-net/SONiC/pull/1699 PR

Reply:
Yes, I agreed and will move this info to platform.json

Reply:
This is still not updated